Makefile

```diff
@@ -16,21 +16,23 @@ compile_messages:
 
 .PHONY: lint_python
 lint_python:
-	flake8 .
-	@# TODO(skarzi): fix type hitings and require `mypy` to pass
+	ruff .
+	@# TODO(skarzi): fix type hints and require `mypy` to pass
 	mypy . || true
-	bandit --configfile pyproject.toml --recursive .
-	pydocstyle src/wily
-	find . -type f -name '*.py' | xargs pyupgrade --py37-plus
 
 .PHONY: lint_formatting
 lint_formatting:
 	black --check .
-	isort --check-only .
 
 .PHONY: lint_spelling
 lint_spelling:
-	codespell || true
+	# TODO(cclauss): Why is codespell not reading config from pyproject.toml or setup.cfg?
+	# 1st command says tomli: command not found ; 2nd command says Requirement already satisfied!!
+	tomli --version || pip install --upgrade tomli
+	codespell --config setup.cfg || true  # Fails!
+	codespell --toml pyproject.toml || true  # Fails!
+	# At least codespell config on the command line works as expected...
+	codespell check-filenames=True ignore-words-list=fo,wil skip="./.*,*.po,*.svg"
```

Comment:
Line 31 is a mystery to me.  `tomli` can not be found but is already installed.  Why is `codespell` accessible but `tomli` not?

Even when we bypass `tomli` on line 32, `codespell` is unable to read its config from `setup.cfg`.

I have not used `flit` before.
* Do we need a [`flit installl` or `flit init`](https://flit.pypa.io/en/stable/cmdline.html#flit-install) to be inside the environment that contains `tomli`?
* Is the presence of `tomli-w` somehow interfering with `tomli`?
* Does `make` have some mechanism for blocking access to executables like [`tox`'s `allowlist_externals`](https://tox.wiki/en/latest/config.html#allowlist_externals)?

Reply:
Fixed with `"'codespell[toml]'~=2.2.5",`